examples/simple/main.go

```diff
@@ -2,14 +2,14 @@ package main
 
 import (
 	"fmt"
+	transactionhandler "github.com/bitcoin-sv/arc/pkg/api/transaction_handler"
```

Comment:
No need for alias as rule has been excluded

Reply:
Removed